### PR TITLE
Phantom deeplinks

### PR DESCRIPTION
**Pull request by phoenixVS**
# Breaking Changes

N/A

# Changes

- feat:
- fix:
- chore:

# Associated Issues

closes #...

### CODE REVIEW

File: packages/scaffold/src/views/w3m-connect-view/index.ts [hunk 1]
```diff
@@ -145,7 +145,7 @@ export class W3mConnectView extends LitElement {
 
   private announcedTemplate() {
     return this.connectors.map(connector => {
-      if (connector.type !== 'ANNOUNCED' || CoreHelperUtil.isMobile()) {
+      if (connector.type !== 'ANNOUNCED') {
```
[the remark this answers is not in the document]
Comment (svenvoskamp) [reply packages/scaffold/src/views/w3m-connect-view/index.ts:148]: We removed this check in the hotfix this morning, this was indeed causing issues that announced wallets didn't show up

**Review by svenvoskamp — CHANGES_REQUESTED** [review]
Could you please fix the merge conflicts? 

We only need to the changes from `src/utils/defaultConfig.ts` and `solana/src/client.ts`

**Comment by enesozturk** [comment]
> Could you please fix the merge conflicts?
> 
> We only need to the changes from `src/utils/defaultConfig.ts` and `solana/src/client.ts`

@phoenixVS

File: apps/laboratory/src/components/Solana/SolanaSignTransactionTest.tsx [hunk 2]
```diff
@@ -13,7 +13,7 @@ import { useWeb3ModalAccount, useWeb3ModalProvider } from '@web3modal/solana/rea
 import { solana } from '../../utils/ChainsUtil'
 import { useChakraToast } from '../Toast'
 
-const PHANTOM_DEVNET_ADDRESS = 'EmT8r4E8ZjoQgt8sXGbaWBRMKfUXsVT1wonoSnJZ4nBn'
+const PHANTOM_DEVNET_ADDRESS = '8vCyX7oB6Pc3pbWMGYYZF5pbSnAdQ7Gyr32JqxqCy8ZR'
```
Comment (enesozturk) [inline apps/laboratory/src/components/Solana/SolanaSignTransactionTest.tsx:16]: Where do we get these addresses? @phoenixVS

**Review by enesozturk — APPROVED** [review]

**Review by svenvoskamp — APPROVED** [review]

File: apps/laboratory/src/components/Solana/SolanaSignTransactionTest.tsx (hunk 2 above)
Comment (phoenixVS) [reply apps/laboratory/src/components/Solana/SolanaSignTransactionTest.tsx:16]: No that is just a wallet for an example of a transaction
Can be anything. I changed to one I have access to so I can more easier debug trx workflow

**Comment by phoenixVS** [comment]
> Could you please fix the merge conflicts?
> 
> We only need to the changes from `src/utils/defaultConfig.ts` and `solana/src/client.ts`

Fixed, only added a few to laboratory test small change
It's just a test wallet address, so I've changed it to one I have access to and debug transactions easier

File: packages/solana/src/client.ts [hunk 3]
```diff
@@ -324,6 +347,8 @@ export class Web3Modal extends Web3ModalScaffold {
   private syncConnectors() {
     const w3mConnectors: Connector[] = []
 
+    syncInjectedWallets(w3mConnectors, this.walletAdapters)
```
[the remark this answers is not in the document]
Comment (phoenixVS) [reply packages/solana/src/client.ts:350]: No, same if on bottom, I think I would revert this change

File: packages/solana/src/client.ts [hunk 4]
```diff
@@ -255,9 +255,32 @@ export class Web3Modal extends Web3ModalScaffold {
...
+          this.checkActiveProviders()
+          this.syncConnectors()
+        } else {
+          setTimeout(() => {
+            checkWallet()
+          }, timer)
+        }
+      }
+
+      setTimeout(() => {
```
[the remark this answers is not in the document]
Comment (phoenixVS) [reply packages/solana/src/client.ts:281]: The problem is Brave, sometime doesn't add [solflare|trust|backpack] objects to Window for some delay
Haven't found any sources, but left this workaround for now

I will left more detailed explanation in comment

**Comment by enesozturk** [comment]
> I will left more detailed explanation in comment

@phoenixVS Please add some sources

**Pull request merged**